src/client/components/app-core/crisp-css-override.css

```diff
@@ -0,0 +1,72 @@
+@import '../app-core/variables.css';
```

Comment:
Please move to `src/clients/components/lazy-chat/providers/crisp.css` (next to the JS file required for Crisp)